.gitlab-ci.yml

```
@@ -998,3 +998,9 @@ plugin:ci-tactician:
 
 plugin:ci-waterproof:
   extends: .ci-template-flambda
+
+plugin:ci-trakt:
+  extends: .ci-template-flambda
+  needs:
+  - build:edge+flambda
+  - plugin:ci-elpi
```

```suggestion
  - plugin:ci-elpi_hb
```
there's no elpi only job